modular_skyrat/modules/fishing/code/fishing.dm

```diff
@@ -9,6 +9,18 @@ GLOBAL_LIST_INIT(fishing_weights, list(
 	/obj/item/xenoarch/strange_rock = 5,
 ))
 
+#define TRAIT_FISHING_MASTER "fishing_master"
+
+/obj/item/skillchip/fishing_master
+	name = "Fishing Master skillchip"
+	desc = "A master of fishing, capable of wrangling the whole ocean if we must."
+	auto_traits = list(TRAIT_FISHING_MASTER)
+	skill_name = "Fishing Master"
+	skill_description = "Master the ability to use fish."
+	skill_icon = "certificate"
+	activate_message = "<span class='notice'>The fish and junk become far more visible beneath the surface.</span>"
+	deactivate_message = "<span class='notice'>The surface begins to cloud up, making it hard to see beneath.</span>"
+
 /turf/open
```

Comment:
:-1: Move all of this fishing crap to be a component.

Reply:
It is not 2013. We do not define shit on base classes like this anymore.

Reply:
https://github.com/tgstation/tgstation/blob/master/code/datums/components/README.md